hoot-core/src/main/cpp/hoot/core/io/BulkInsert.h

```diff
@@ -22,7 +22,7 @@
  * This will properly maintain the copyright information. DigitalGlobe
  * copyrights will be updated automatically.
  *
- * @copyright Copyright (C) 2015 DigitalGlobe (http://www.digitalglobe.com/)
+ * @copyright Copyright (C) 2016 DigitalGlobe (http://www.digitalglobe.com/)
```

Comment:
Are you sure we want to replace `2015` with `2016` and not `2015, 2016`?  I'm just going to leave this here for all of the cases like this.  There are a few more below.